Assets/Localization/StringTables/Internal_RSC_en.asset

```diff
@@ -8599,7 +8599,7 @@ MonoBehaviour:
       you ready for %hol? Don''t wait too long.[/record]
 
       %hol is almost here.
-      I wouldn''t wait to the last minute to get ready.[/end]'
+      I wouldn''t wait til the last minute to get ready.[/end]'
```

Comment:
Any specific reason why til wasn't rendered as 'til here?

Reply:
From [The Free Dictionary:](https://www.thefreedictionary.com/till)

>[Middle English, from Old English til, from Old Norse.]
 Till is actually the older word, with until having been formed by the addition to it of the prefix un-, meaning "up to." In the 1700s, the spelling 'till became fashionable, as if till were a shortened form of until. Although 'till is now nonstandard, 'til is sometimes used in this way and is considered acceptable, though it is etymologically incorrect.

Thus this is the proper Old English spelling.

Reply:
Then allow me to ask: Why are we deferring to Old English? Daggerfall doesn't use any Old English spellings of words. There is no "hƿæt" in place of "what" or "hȳran" instead of "hear".

If 'til is undesirable, till seems like the obvious option, not til.